sqs/src/test/java/docs/javadsl/SqsAckTest.java

```diff
@@ -97,15 +82,53 @@ public void testAcknowledgeViaFlow() throws Exception {
               return new CompletableFuture<DeleteMessageResult>();
             });
 
-    // #flow-ack
-    MessageAction action = MessageAction.delete(new Message().withBody("test-ack-flow"));
+    List<Message> messages = createMessages();
+    Source<Message, NotUsed> source = Source.fromIterator(() -> messages.iterator());
+
     CompletionStage<Done> done =
-        Source.single(action)
+        // #ack
```

Comment:
These `#ack`s seem duplicated from line 47/52.

Reply:
I just moved the code down to have the setting tests before the stream tests.